src/main/java/com/websafe/ccmsbe/controller/PrivacyRegulationController.java

```diff
@@ -0,0 +1,58 @@
+package com.websafe.ccmsbe.controller;
+
+
+import com.websafe.ccmsbe.entity.PrivacyRegulation;
+import com.websafe.ccmsbe.service.PrivacyRegulationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin("http://localhost:3000")
+@RequestMapping("api/v1/privacyRegulation")
+public class PrivacyRegulationController {
+
+    private final PrivacyRegulationService privacyRegulationService;
+
+    @Autowired
+    public PrivacyRegulationController(PrivacyRegulationService privacyRegulationService) {
+        this.privacyRegulationService = privacyRegulationService;
+    }
+
+    @PostMapping("/create")
```

Comment:
Let's not use the operation names (create, get, update, delete etc) in the endpoint path. The HTTP request type already denotes that. 

Please change the other places as well.

```suggestion
    @PostMapping("/")
```